### ISSUE TITLE
Code Editor Field Not Working With Latest ACF Beta

### ISSUE THREAD

**Issue by Enchiridion** [acf-extended/ACF-Extended#44]
**Describe the bug**
While testing the latest ACF Pro beta version 5.9.0-beta1, the 'acfe_code_editor' field fails to initialize Codemirror, only a textarea is shown.  No errors are present in the console.

**To Reproduce**
Update ACF to 5.9.0-beta1.

**Expected behavior**
The code editor to appear.

**WordPress & ACF**
WordPress version: 5.4.1
ACF Pro version: 5.9.0-beta1
ACF Extended: 0.8.6.1

**Comment by Enchiridion** [comment]
I did a little more testing and found ACF's native Image field is also not working on the latest beta, so there may not be anything ACFE needs to update.  I'll test this again when ACF releases another beta.

**Comment by acf-extended** [comment]
Hello,

Thanks for the feedback! I'll check to see what's going on with the Editor field.
The image field is working correctly on my side (both Media & Basic "Uploader Type").

Regards.

**Comment by acf-extended** [comment]
Hello,

Okay I found the problem. It comes from ACF Pro 5.9-beta2 which removed the `do_action('acf/enqueue_scripts');` from the codebase. This is the hook that is used to enqueue all fields JS (not only Code Editor). What is weird is that the hook is still used (but without doing anything) in the ACF Pro 5.9-beta2 codebase.

In the file `/advanced-custom-fields-pro/includes/media.php:26` it is called like that:

```php
add_action('acf/enqueue_scripts', array($this, 'enqueue_scripts'));
```

It looks like an oversight. I've opened a ticket on the ACF support to let them know. I'll wait for their answer to know if I should change hook or not.

Have a nice day!

Regards.
